CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+YaST Contribution Guidelines
```

Comment:
yast organization uses shared contributing, so if it will be under it, it need not this file: https://github.com/yast/.github

Reply:
Replaced file in the commit.